src/platform/intel/cavs/platform.c

```diff
@@ -482,17 +480,17 @@ int platform_init(struct sof *sof)
 	trace_point(TRACE_BOOT_PLATFORM_IPC);
 	ipc_init(sof);
 
-	/* init DAIs */
-	ret = dai_init();
-	if (ret < 0)
-		return ret;
-
 	/* initialize IDC mechanism */
 	trace_point(TRACE_BOOT_PLATFORM_IDC);
 	ret = idc_init();
 	if (ret < 0)
 		return ret;
 
+	/* init DAIs */
+	ret = dai_init();
+	if (ret < 0)
+		return ret;
+
```

Comment:
Did you actually move dai_init() later in the code? Or is git diff broken?

Reply:
Yes, I did.

Reply:
@tlauda Ok, I understand. Then please add the change in a separate patch and explain why the movement was necessary.

Also, please make sure that the code compiles after each patch so that we won't break bisection in case of problems.

Reply:
@dbaluta It's because we are changing ordering of the trace points as already stated in the commit. I understand that we should have as much granularity as possible, but I feel it's an overkill to split that kind of simple change into 4 separate commits.

Reply:
@tlauda ok, no strong preference on this change. Lets hope people will understand the reason behind this change from the commit message :).